ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -437,6 +437,20 @@
     "EndButtonLabel": "Save and continue",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/update-task-list/submitted"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "changeCaseName-superuser",
+    "Name": "Change case name",
+    "Description": "Change case name",
+    "DisplayOrder": 15,
```

Comment:
does it matter that this `DisplayOrder` value is already used, or is it intentional?